rotkehlchen/chain/ethereum/adex/utils.py

```diff
@@ -0,0 +1,130 @@
+)
+STAKING_ADDR = '0x4846C6837ec670Bbd1f5b485471c8f64ECB9c534'
+CREATE2_SALT = f'0x{bytearray(32).hex()}'
+ADX_AMOUNT_MANTISSA = 10**18
+
+# Pools data
+TOM_POOL_ID = HexStr('0x2ce0c96383fb229d9776f33846e983a956a7d95844fac57b180ed0071d93bb28')
+POOL_ID_POOL_NAME = {
+    TOM_POOL_ID: 'Tom',
+}
+
+# Tom pool fee rewards API constants
+TOM_POOL_FEE_REWARDS_API_URL = 'https://tom.adex.network/fee-rewards'
+SECONDS_PER_YEAR = FVal('31536000')
```

Comment:
Why Fval? It's an int. Also it's already defined as `YEAR_IN_SECONDS` in `constants/timing.py`